<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>libToolWidget</class>
 <widget class="QWidget" name="libToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="BTopenPath">
       <property name="text">
        <string>openPath</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTrun">
       <property name="text">
        <string>run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTclear">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Release</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTreleaseCopy">
       <property name="text">
        <string>Copy Release</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="relaseText"/>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Debug</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTdebugCopy">
       <property name="text">
        <string>Copy Debug</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="debugText"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTextEdit" name="textEditLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
